.gitlab-ci-full-integration-template.yml

```diff
@@ -1,4 +1,7 @@
 # The CI job name will be unique after substitution from the generator.
+include:
+  - remote: 'https://raw.githubusercontent.com/mendersoftware/mendertesting/master/qa-common/send-results-to-mantra.yml'
```

Comment:
Two comments here:

1. We only need **one** `include` and it should be here, not in the generator. It could also be in the main gitlab-ci file for better transparency, but that I think is a matter of taste.

2. Can we please use the standard syntax like in the main gitlab-ci file? For consistency 🙏 I also think that it reads better than the remote one, although this you may say that it is also a matter of taste.
```
include:
  - project: "Northern.tech/Mender/mendertesting"
    file: "qa-common/send-results-to-mantra.yml"
```

You can add ref if you want:
```
include:
  - project: "Northern.tech/Mender/mendertesting"
    file: "qa-common/send-results-to-mantra.yml"
    ref: "master"
```